src/v/raft/vote_stm.cc

```diff
@@ -268,8 +275,8 @@ ss::future<> vote_stm::update_vote_state(ssx::semaphore_units u) {
           ec.value(),
           ec.message());
     } else {
-        vlog(_ctxlog.info, "became the leader term: {}", term);
         if (term == _ptr->_term) {
```

Comment:
Looks like we don't need this check any more

Reply:
Actually no, I think I am wrong, the term may increase due to AppendEntries replies that we receive when trying to replicate the configuration. So the code is correct.

Reply:
were you referring to not needing the check because of the new boolean check added up above, but then saying that because of `    auto ec = co_await replicate_config_as_new_leader(std::move(u));` that assumption may become invalid?

Reply:
yes exactly